### PR TITLE
Added Additional properties table to all db scripts.

### PR DESCRIPTION
**Pull request by kavishkafernando**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/oracle.sql [hunk 1]
```diff
@@ -434,6 +434,17 @@ CREATE TABLE AM_API_DOC_META_DATA (
...
 /
 
+CREATE TABLE AM_API_ADDITIONAL_PROPERTIES (
+  UUID VARCHAR2(255),
+  API_ID VARCHAR2(255),
+  PROPERTY_KEY VARCHAR2(255),
+  PROPERTY_VALUE VARCHAR2(255),
+  UNIQUE (UUID),
+  PRIMARY KEY (API_ID, PROPERTY_KEY),
+  FOREIGN KEY (API_ID) REFERENCES AM_API(UUID)  ON DELETE CASCADE
```
Comment (mushthaq33) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/oracle.sql]: Additional space before ON

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/mssql.sql [hunk 2]
```diff
@@ -303,6 +303,16 @@ CREATE TABLE AM_API_RESOURCES (
   FOREIGN KEY (RESOURCE_CATEGORY_ID) REFERENCES AM_RESOURCE_CATEGORIES(RESOURCE_CATEGORY_ID)
 );
 
+CREATE TABLE AM_API_ADDITIONAL_PROPERTIES (
+  UUID VARCHAR(255),
+  API_ID VARCHAR(255),
+  PROPERTY_KEY VARCHAR(255),
+  PROPERTY_VALUE VARCHAR(255),
+  UNIQUE (UUID),
+  PRIMARY KEY (API_ID, PROPERTY_KEY),
```
Comment (uvindra) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/mssql.sql]: Any reason API_ID, PROPERTY_KEY being PRIMARY KEY? These should be unique constraints. UUID seems to be the PRIMARY_KEY for this table

Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/mssql.sql]: fixed

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/oracle.sql (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/resources/dbscripts/oracle.sql]: fixed

**Pull request merged**